frappe_blurhash/utils/get_blur_hash.py

```diff
@@ -0,0 +1,30 @@
+import frappe
+
+
+def get_blur_hash(dt, dn, df):
+    """
+    dt : Doc-type
+    dn : Doc-name
+    df : Doc-field
+    """
+    file_url = frappe.get_value(dt, dn, df)
+    if frappe.is_table(dt):
+        dt, dn = frappe.get_value(dt, dn,
+                                  ("parenttype", "parent"))
+    attachments = get_attachments_by_docfield(dt, dn, df, file_url)
+    return attachments[0].get("blurhash") if len(attachments) else None
+
+
+def get_attachments_by_docfield(dt, dn, df, file_url):
+    if "frappe_blurhash" not in frappe.get_installed_apps():
+        return []
+    fields = ["blurhash"]
```

Comment:
Having this perimeter of this function, and moving this function in its own file as a utility would make sense too.

Reply:
done

Reply:
This function is not in a separate file tho.. (under utils) the rest look fine..